bugzilla/_cli.py

```diff
@@ -185,6 +185,20 @@ def _parser_add_output_options(p):
                  "section 'Output options' for more details.")
 
 
+def _parser_add_field_passthrough_opts(p):
+    # Put this at the end, so it sticks out more
```

Comment:
By moving these lines into a separate function, this comment looses its meaning. Maybe it can be removed?

```suggestion
```